wifiphisher.py

```diff
@@ -337,33 +337,13 @@ def reset_interfaces():
             Popen(['ifconfig', m, 'up'], stdout=DN, stderr=DN)
 
 
-def get_internet_interface():
-    '''return the wifi internet connected iface'''
+def interface_or_ip_prefix(iface=None, prefix=None):
```

Comment:
What if both iface and prefix are True or both are None? If these are valid cases, we need to document them.